### DISCUSSION TITLE
interactive flowmap

### DISCUSSION THREAD

**Discussion by rhs94** [walkerke/mapgl#9] [Ideas]
Hi,
I just saw your post in linkedin about this library. I will see how I can use it for my work as I really began to love and use geo data in R during the last year. What I use most is flowmaps. Most often I use the github library FlowmapBlue/flowmapblue.R which is also awesome. Is it possible that you could implement something like that? I'm asking because you wrote that you will submit it to CRAN and that's what I really need.. I most often have issues when using remote packages. Thanks in advance!

**Comment by e-kotov** [comment]
related #37
